doc/source/man.rst

```diff
@@ -15,36 +15,38 @@ Synopsis
 Description
 -----------
 
-rho is a network discovery tool to identify the number of systems running on a network, their operating system and versions of some key packages and products for almost any Linux or Unix flavor. Being able to identify the systems running on the network is a vital component to managing entitlements (licenses and renewals). Ultimately, discovery is part of the larger sysadmin task of managing inventories.
+Rho, and the rho command, is a network discovery tool to identify environment data, or *facts*, such as the number of physical and virtual systems on a network, their operating systems and other configuration data, and versions of some key packages and products for almost any Linux or UNIX version. The ability to discover and identify the software and systems that are running on the network improves your ability to understand and report on your entitlement usage. Ultimately, this discovery is part of the larger system administration task of managing your inventories.
```

Comment:
perhaps:

> ... is a network discovery and inspection tool ...


and 

> ... Ultimately, this discovery and inspection is part of the larger ...

I feel like discovery is just locating and connecting to the system and the inspection part is where we find out about the system.